src/configuration/index.ts

```diff
@@ -0,0 +1,24 @@
+import {Network, NetworkConfiguration} from "../network/interfaces";
+import {Wallet} from "../interfaces";
+import {defaultConfiguration, getNetworkConfiguration} from "../network";
+
+export interface Configuration {
```

Comment:
Why do you have this interface?

Reply:
My reasoning was if we ever needed any kind of additional data inside configuration we could expand this `Configuration` interface (with a new "category of configs") and additionally it allows me this definition `Network | NetworkConfiguration` which I find handy.

If you think that we won't extend configuration object I can remove this interface and just expand `NetworkConfiguration` with:

```
network?: Network
```

Reply:
I think structure of plugin state should be:
```javascript
{
   polkadot: {
      config: SnapConfig {
          network,
          unit,
          //other stuff 
     },
    account: Account
   }
}
```

Reply:
where we have default SnapConfig, which properties can be overriden